examples/graphene_apps/python_worker/ov_workload/openvino.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python3
+
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+import logging
+import json
+import toml
+from avalon_worker.workload.workload import WorkLoad
+from avalon_worker.utility.zmq_comm import ZmqCommunication
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler(sys.stdout))
+
+# -------------------------------------------------------------------------
+
+
+class OpenVinoWorkLoad(WorkLoad):
+    """
+    OpenVino workload class. This is an example workload.
+    """
+    def __init__(self):
+        """
+        Constructor for OpenVinoWorkLoad.
+        """
+        # Zmq socket used for communicating with Openvino C++ workload.
+        # Read zmq url from config file.
+        try:
+            config = toml.load("ov_workload_config.toml")
+        except Exception as e:
+            logger.error("Unable to read ov_workload_config.toml" + str(e))
+            raise
+        zmq_url = config.get("zmq_url")
+        self.zmq_socket = ZmqCommunication(zmq_url)
+        self.zmq_socket.connect()
+        super().__init__()
+
+# -------------------------------------------------------------------------
+
+    def execute(self, in_data_array):
```

Comment:
Is this function executing work order? This is function is constructing the request and sending over ZMQ. Change the function name to proper name.

Reply:
This is an function in abstract class (WorkLoad) which is overridden here. It's an existing class